fhf-datafangst-client/package-lock.json

```diff
@@ -21474,6 +21507,15 @@
       "version": "0.1.13",
       "license": "Apache-2.0"
     },
+    "node_modules/tsc": {
+      "version": "2.0.4",
+      "resolved": "https://registry.npmjs.org/tsc/-/tsc-2.0.4.tgz",
+      "integrity": "sha512-fzoSieZI5KKJVBYGvwbVZs/J5za84f2lSTLPYf6AGiIf43tZ3GNrI1QzTLcjtyDDP4aLxd46RTZq1nQxe7+k5Q==",
+      "dev": true,
+      "bin": {
+        "tsc": "bin/tsc"
+      }
+    },
```

Comment:
You should never edit the package-lock manually, only through `npm` commands.
i.e. `npm install` adds dependencies and `npm uninstall` removes dependencies (or running `npm install` after modifying package.json).

In this case, since you need to run `npm install` to update the package-lock